src/affiliations/static/css/hide_admin_original.css

```diff
@@ -0,0 +1,7 @@
+td.original {
```

Comment:
I would add a comment that explains why this stylesheet exists.

Reply:
Good idea, I have just added that.